rskj-core/src/test/java/co/rsk/remasc/RemascTestRunner.java

```diff
@@ -215,11 +212,11 @@ public static Block createBlock(Block genesis, Block parentBlock, Sha3Hash block
 
         byte[] diffBytes = BigInteger.valueOf(difficultyAsLong).toByteArray();
 
-        BigInteger paidFees = BigInteger.ZERO;
+        Coin paidFees = Coin.ZERO;
         for (Transaction tx : txs) {
-            BigInteger gasLimit = BigIntegers.fromUnsignedByteArray(tx.getGasLimit());
-            BigInteger gasPrice = BigIntegers.fromUnsignedByteArray(tx.getGasPrice());
-            paidFees = paidFees.add(gasLimit.multiply(gasPrice));
```

Comment:
It's safe to remove the unsigned byte array function?

Reply:
`fromUnsignedByteArray` is implemented as `return new BigInteger(1, buf)`. I used the wrong constructor but I'll change it!

Reply:
:+1: